From 8e042105b7267a7f495547eacd304dd65664e295 Mon Sep 17 00:00:00 2001
From: Francois Daoust <fd@tidoust.net>
Date: Mon, 19 Sep 2022 17:10:41 +0200
Subject: [PATCH] Add IDL patch for Web Animations

The `AnimationTimeline` interface is (temporarily) re-defined in Scroll
Animations and will be merged in Web Animations. No specific tracking issue
(issue appears as a note in the Scroll Animations ED) but this patch will fail
as soon as the IDL in Web Animations gets updated, so we'll know when the patch
can be dropped.
---
 ed/idl/web-animations.idl | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/ed/idl/web-animations.idl b/ed/idl/web-animations.idl
index 1b078e6ea..07df90672 100644
--- a/ed/idl/web-animations.idl
+++ b/ed/idl/web-animations.idl
@@ -3,11 +3,6 @@
 // (https://github.com/w3c/webref)
 // Source: Web Animations (https://drafts.csswg.org/web-animations-1/)
 
-[Exposed=Window]
-interface AnimationTimeline {
-    readonly attribute double? currentTime;
-};
-
 dictionary DocumentTimelineOptions {
   DOMHighResTimeStamp originTime = 0;
 };
-- 
2.36.0.windows.1

